package/pubspec.yaml

```diff
@@ -10,9 +10,11 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  geolocator: ^9.0.2
   collection: ^1.15.0
   flutter_svg: ^2.0.2
+  geolocator: ^10.0.0
```

Comment:
@Allen7702 what will be function for latlong2 packages? If for get distances, then geolocator can do the trick for you. See below screenshop

<img width="805" alt="image" src="https://github.com/hisptz/dhis2-flutter-ui/assets/4928914/a9e4a59b-1f2f-42c8-964c-2570008e4bc0">

Reply:
In the process of displaying the map, i used the flutter_map package, this package uses latlong2 as one of it's dependencies, of which it gets used when trying to mark a point on the map. 
Lakini naweza iondoa since by default the the flutter_map package tayari ina latlong2 inside